src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+function Person({ person }) {
+  return (
```

Comment:
make a destructuring here
```suggestion
function Person({ person }) {
  const { ... } = person;
  return (
```